docs/apm-data-security.asciidoc

```diff
@@ -152,31 +194,32 @@ Certain agents offer additional filtering and obfuscating options:
 **Agent configuration options**
 
 * (Node.js) Remove errors raised by the server-side process:
-Disable with {apm-node-ref-v}/configuration.html#capture-exceptions[captureExceptions].
+disable with {apm-node-ref-v}/configuration.html#capture-exceptions[captureExceptions].
 
 * (Java) Remove process arguments from transactions:
-* Disabled by default with {apm-java-ref-v}/config-reporter.html#config-include-process-args[`include_process_args`].
+disabled by default with {apm-java-ref-v}/config-reporter.html#config-include-process-args[`include_process_args`].
 
-[discrete]
-[[custom-filter]]
-=== Custom filters
+// ****************************************************************
 
-There are two ways to filter or obfuscate other types of APM data:
+[[custom-filter]]
+==== Custom filters
 
-* <<filters-ingest-pipeline>>
-* <<filters-in-agent>>
+include::./apm-data-security.asciidoc[tag=custom-filters]
 
 [discrete]
 [[filters-ingest-pipeline]]
 ==== Create an ingest node pipeline filter
 
 Ingest node pipelines specify a series of processors that transform data in a specific way.
-Transformation happens prior to indexing–inflicting no performance overhead on the monitored application.
+Transformation happens prior to indexing--inflicting no performance overhead on the monitored application.
 Pipelines are a flexible and easy way to filter or obfuscate Elastic APM data.
 
+WARNING: Changes to the default APM pipeline do not persist through upgrades.
+You will need to re-add custom ingest pipelines after every upgrade.
```

Comment:
How do we feel about documenting this workaround? I know in the past we said we didn't want to do this (https://github.com/elastic/apm-server/pull/6940#discussion_r782735208), but I don't see another option.

Reply:
Hmmm. It looks like config changes also overwrite the pipeline. I'm not sure how to move forward here.

Reply:
I don't see any other option than documenting that custom ingest pipelines are not supported in the integration at this point. There are plans to add this functionality to the Fleet eco system, but until it lands there's nothing much we can do. 

> Hmmm. It looks like config changes also overwrite the pipeline.

@joshdover is this on purpose?

Reply:
Okey dokey. I'll comment out all of the ingest pipeline examples for now.

Reply:
All ingest node content removed in [`5b24889`](https://github.com/elastic/apm-server/pull/8043/commits/5b24889e7f15a5c8f9e2dfc16dce7c960272dab5).